drivers/scheduler/k8s/k8s.go

```diff
@@ -3182,6 +3192,71 @@ func (k *K8s) GetVolumes(ctx *scheduler.Context) ([]*volume.Volume, error) {
 	return vols, nil
 }
 
+// GetPureVolumes  Get the Pure volumes (if enabled) by type (PureFile or PureBlock)
```

Comment:
Do we have a better way to get all pure volumes instead of creating a new volume list?

Reply:
sry, what we mean by a new volume list

Reply:
the function I added is a variation of an existing getVolumes function, as far as I know this is the only way to get all volumes, what's the concern with the list.